apps/mark-scan/backend/src/util/render_ballot.tsx

```diff
@@ -33,16 +35,22 @@ export interface RenderBallotProps {
 
 const MACHINE_TYPE: MachineType = 'markScan';
 
-function getPaperDimensions(): PaperDimensions {
-  /* istanbul ignore next - hardware support in flux - @preserve */
-  return getMarkScanBmdModel() === 'bmd-150'
-    ? PAPER_DIMENSIONS['Bmd150']
-    : PAPER_DIMENSIONS['Letter'];
+function getPaperDimensions(election: Election): PaperDimensions {
+  if (getMarkScanBmdModel() === 'bmd-150') {
```

Comment:
I briefly considered removing all the old VSAP 155 logic, but that quickly ballooned so I punted